.github/workflows/build-and-deploy-container-image.yaml

```
@@ -224,7 +224,28 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 60
 
+    services:
+      npq:
+        image: registry.neohelden.com/neohelden/nats:2.6.3-2.0.0
+        credentials:
+          username:  ${{ secrets.image-registry-username }}
+          password: ${{ secrets.image-registry-token }}
+        ports:
+          - 8222:8222
+          - 4222:4222
+    env:
+      # Dont judge me https://github.com/nats-io/nats-docker/blob/4fe4d656443e2a43caab61ecd665436e12e3358a/2.2.6/alpine3.13/nats-server.conf
```

We should change this
```suggestion
      # Dont judge me https://github.com/neohelden/nats/blob/eee9e594d6b451c43192ba05f5cb0da609a8b38d/Dockerfile#L5
```